backend/matchmaking/consumers.py

```
@@ -95,6 +97,12 @@ async def match_player(self):
             player_in_queue: WaitingPlayer | None = await self.get_player_in_queue(player_to_match)
             matched_user: WaitingPlayer | None = await self.get_matched_user(player_to_match)
 
+            player_in_queue_username = await player_in_queue.get_username()
+            matched_user_username = await matched_user.get_username()
+            
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)